gluten-substrait/src/main/java/org/apache/gluten/substrait/plan/PlanBuilder.java

```diff
@@ -63,7 +63,8 @@ public static PlanNode makePlan(
       TypeNode outputSchema,
       AdvancedExtensionNode extension) {
     if (subCtx == null) {
-      throw new NullPointerException("ColumnarWholestageTransformer cannot doTansform.");
+      throw new NullPointerException(
+          "Cannot execute doTransform due to the SubstraitContext is null.");
```

Comment:
Could remove the trailing dot in error message, by the way.

Also, normally we could do `Preconditions.checkNotNull(...)` to simplify.

Reply:
Sounds good to me. Let's using `Preconditions.checkNotNull(...)` instead.